src/essays/graph-experience-learning/index.md

```diff
@@ -0,0 +1,147 @@
+"guacamole is what they like"
+
+![graphy chart]
+"they're in the mood for something savory"
+
+As you're trying to abstract why a bunch of things are connected, you can use categories or tags or groupings of any kind. Metadata drives what surprises and delights users. If this simple example was fed back to the system, later a user filtering or searching for savory flavors may have never considered campfire, and say "whoa!"
+
+We can't do much with the fact that guacamole is popular other than keep making more of the same product.
+
+As more specific groupings are formed by way of curation, we arrive at a dataset that informs the how and why to selections made. What's more, when a database that understands Graphy Data is used (Arango, Neo4j), each connection or "edge" can be a composite of datums that are tied to their creators, timestamped, disabled or promoted.
+
+For example, Jane Doe might be super popular, and your customer base might benefit from her personal curations having more weight than normal.
+
+Those factors are important to record because ideas get stale, people make messes, and we need machine unlearning as a species perhaps more than we need machines to assist us in learning. The most successful models I believe will be ones that expand the fringe, revisit truths as potentially trendy, and constantly invite humans to participate in strengthening or weakening those connections.
+
+I'm aiming someone that's interested in computers maybe, but I definitely want to define all the stuff, because anyone looking at this is likely a manager wanting to know if this is good for their business ideas, someone who is in the traditional learning camp. Right now machine learning and AI look scary, and bad, and will take our jobs. I'm trying to paint a picture where AI is a new tool as revolutionary as the pencil with the eraser on the end. I think it's a misnomer that we'll make that progress while telling people that they won't need to do anything different, and we'll simply figure out how to augment.
+
+There's very specific cases where asking for feedback works, and one should still employ a person to decide where the computer should be asking the questions. 
+
+
+
+
+# {{ $page.title }} (<em>{{ $page.frontmatter.title_sub }}</em>)
+I've realized something profound. The facts themselves are fairly sound, it might just be our beliefs and expectations that are delicate. "People are the problem" as they say. The realization is that the following three things are extremely similar in nature
+
+* user experience design
+* machine learning
+* graph databases
+
+## The Color of Food
+One size/flavor does not fit all. Howard Muskowitz made that abundantly clear in the 1970s. If he had been mistaken, we would never have had chunky tomato sauce. Over the years I've come to refer to the color sections of stylesheets as "flavors" There's a lot more flexibility to shift the grade of your particular shades of white when you name your colors after foods.
```

Comment:
Perhaps a citation supporting this and some info about who `Howard Muskowitz` is.